recipe/meta.yaml

```diff
@@ -1,21 +1,22 @@
-{% set version = "0.3.0" %}
+{% set version = "1.0.0.dev" %}
 
 package:
   name: geopandas
   version: {{ version }}
 
 source:
-  url: https://github.com/geopandas/geopandas/releases/download/v{{ version }}/geopandas-{{ version }}.tar.gz
-  sha256: e63bb32a3e516d8c9bcd149c22335575defdc5896c8bdf15c836608f152a920b
```

Comment:
How do we update builds in either of the following cases:

1.  geopandas master (without cython) wants to release?  

    We revert these changes?
2.  geopandas geopandas-cython wants to push a new dev build?
    
    We just update the sha here?  Perhaps also bump the dev build number?

Reply:
> geopandas master (without cython) wants to release?

No need to revert but substitute this with the GitHub URL, but if the branch is more up-to-date than this archived release I can put it in there now.

> geopandas geopandas-cython wants to push a new dev build?

I would recommend bumping the build number or, if geopandas uses versioneer, just use that version as a new release.

Reply:
Yes, geopandas uses versioneer, so the last 'version' is 

```
In [2]: geopandas.__version__
Out[2]: '1.0.0.dev+100.g170c235'
```